Resources/Prototypes/_DV/Actions/station_ai.yml

```diff
@@ -0,0 +1,159 @@
+# Actions
+- type: entity
+  id: ActionStationAiOpenShop
+  name: Choose programs
+  description: Configure your installed programs
+  components:
+  - type: InstantAction
+    itemIconStyle: BigAction
+    icon:
+      sprite: _DV/Interface/Actions/actions_ai.rsi
+      state: store
+    event: !type:StationAiShopActionEvent
+
+- type: entity
+  id: ActionStationAiRgbLighting
+  name: ai-rgb-lighting-name
+  description: ai-rgb-lighting-desc
+  components:
+  - type: EntityTargetAction
+    whitelist:
+      tags:
+      - LightFixture
```

Comment:
why doesnt this use LightFixture component which is used below

Reply:
the component shared by wall lights is server-only unfortunately so the client can't use it as a whitelist

Reply:
also typo